### PR TITLE
Update dockerfile style guide

### PR DESCRIPTION
**Pull request by kbergin**
I don't think we should encourage folks to email such a large internal email list with dockerfile questions, I'm proposing that filing an issue would be more appropriate but am open to other ideas!

### CODE REVIEW

**Comment by wjdingman1** [comment]
@kbergin totally agree, but have we actually ever had anybody email about style guide questions?

**Comment by kbergin** [comment]
Not to my knowledge, but I may not be on that email list. Either way I do think we should point folks somewhere else.

**Review by wjdingman1 — APPROVED** [review]

File: dockers/README.md [hunk 1]
```diff
@@ -224,4 +224,4 @@ ENTRYPOINT [ "/sbin/tini", "--" ]
 
 ## <a link="trouble"></a> Troubleshooting
 
-If you have any questions or would like some more guidance on writing Dockerfiles please reach out at [dsde-engineering@broadinstitute.org](dsde-engineering@broadinstitute.org).
+If you have any questions or would like some more guidance on writing Dockerfiles please file a github issue in WARP.
```
Comment (ekiernan) [inline dockers/README.md:227]:
```suggestion
If you have any questions or would like some more guidance on writing Dockerfiles please file a [GitHub issue in WARP](https://github.com/broadinstitute/warp/issues/new).
```

**Review by ekiernan — APPROVED** [review]

**Review by nikellepetrillo — APPROVED** [review]

**Comment by jessicaway** [comment]
add to whitelist

**Pull request merged**